### DISCUSSION TITLE
Default Translation for users

### DISCUSSION THREAD

**Discussion by lukas-schaetzle** [directus/directus#7872] [Archive]
Collections and interfaces allow defining translations for different languages. However, there is no option for a default translation which is used as a fallback when there's no translation available for the currently used language.
This would be beneficial for multi language projects (e.g. admin uses English, editor uses German) where you want to use the same collection names for all users (but not the collection keys which may not always be reader friendly for the editors). Currently this is only possible, if you always register 2 translations with the same target value for every collection which is a bit cumbersome.

**Comment by benhaynes** [comment]
Let's use this ticket to track this request: https://github.com/directus/directus/discussions/3945
